.github/workflows/e2e-suite.yml

```
@@ -99,8 +103,14 @@ jobs:
           mv calicoctl-linux-amd64 /usr/local/bin/calicoctl
           mv kubectl /usr/local/bin/kubectl
 
+      - name: Set OpenAPI Spec URL
+        if: ${{ inputs.openapi_spec_url != '' }}
+        run: |
+          echo "Using OpenAPI Spec URL: ${{ inputs.openapi_spec_url }}"
+          echo "OPENAPI_SPEC_URL=${{ inputs.openapi_spec_url }}" >> $GITHUB_ENV
+
       - name: Install Package
-        run: make install
+        run: make install SPEC="${OPENAPI_SPEC_URL}"
```

One more small thing, I don't think GHA's env is being implicitly passed in here so we may need to reference `env.OPENAPI_SPEC_URL` directly:

```suggestion
        run: make install SPEC="${{ env.OPENAPI_SPEC_URL }}"
```

---

++ on this. Another possible solution is `{{ inputs.openapi_spec_url }}`